src/core/dev_api/openvino/op/paged_attention.hpp

```diff
@@ -1,29 +1,133 @@
+                            const Output<Node>& max_context_len,
+                            const Output<Node>& xattention_threshold,
+                            const Output<Node>& xattention_block_size,
+                            const Output<Node>& xattention_stride);
+
+    /// \brief Constructs a PagedAttentionExtension operation with rotation support. (16 parameter constructor)
+    ///
+    /// \param query                Query tensor.
+    /// \param key                  Key tensor.
+    /// \param value                Value tensor.
+    /// \param key_cache            Cached key tensor.
+    /// \param value_cache          Cached value tensor.
+    /// \param past_lens            Lengths of past sequences.
+    /// \param subsequence_begins   Subsequence start indices.
+    /// \param block_indices        Indices of memory blocks.
+    /// \param block_indices_begins Start indices for block indexing.
+    /// \param scale                (Optional) Scaling factor for attention scores.
+    /// \param sliding_window       (Optional) Sliding window size for local attention.
+    /// \param alibi_slopes         (Optional) ALiBi slopes for biasing attention.
+    /// \param max_context_len      Maximum context length.
+    /// \param rotated_block_indices (Optional) Rotated block indices.
+    /// \param rotation_deltas       (Optional) Rotation deltas.
+    /// \param rotation_trig_lut     (Optional) Rotation trig lookup table.
+    PagedAttentionExtension(const Output<Node>& query,
+                            const Output<Node>& key,
+                            const Output<Node>& value,
+                            const Output<Node>& key_cache,
+                            const Output<Node>& value_cache,
+                            const Output<Node>& past_lens,
+                            const Output<Node>& subsequence_begins,
+                            const Output<Node>& block_indices,
+                            const Output<Node>& block_indices_begins,
+                            const Output<Node>& scale,
+                            const Output<Node>& sliding_window,
+                            const Output<Node>& alibi_slopes,
+                            const Output<Node>& max_context_len,
+                            const Output<Node>& rotated_block_indices,
+                            const Output<Node>& rotation_deltas,
+                            const Output<Node>& rotation_trig_lut,
+                            const Output<Node>& xattention_threshold,
+                            const Output<Node>& xattention_block_size,
+                            const Output<Node>& xattention_stride);
+
     void validate_and_infer_types() override;
     std::shared_ptr<ov::Node> clone_with_new_inputs(const ov::OutputVector& new_args) const override;
 
+    /// \brief Gets the output element type at the specified index.
+    const ov::element::Type get_out_type(int index) const;
```

Comment:
Why is required?
The getters/setters should be for attributes?
is not same as node::get_output_element_type()